frontend_app_admin_workspace_user/src/css/index.styl

```diff
@@ -31,9 +31,12 @@
         white-space nowrap
         overflow hidden
         text-overflow ellipsis
-        cursor pointer
         &:not(:hover)
           color inherit
+        &__link
+          color inherit
+          margin -20px auto
+          padding 20px
```

Comment:
the alignment fix is not necessary, I agree it's better your way but it would require additional fix on other columns.

I'd suggest you to replace the 2 highlighted lines by these lines:
```css
display inline-block
width 100%
```
so that their won't be any regression from the previous code. After that I'll merge your PR.